prod-stack.py

```diff
@@ -79,24 +79,33 @@
         ),
     ])
 
-INFLATION_QUEUES = Sub('ksp=${ksp} ksp2=${ksp2}', ksp=GetAtt(inbound, 'QueueName'), ksp2=GetAtt(inbound2, 'QueueName'))
+INFLATION_QUEUES = Sub('ksp=${ksp} ksp2=${ksp2}', ksp=GetAtt(
+    inbound, 'QueueName'), ksp2=GetAtt(inbound2, 'QueueName'))
```

Comment:
```suggestion
INFLATION_QUEUES = Sub('ksp=${ksp} ksp2=${ksp2}',
                       ksp=GetAtt(inbound, 'QueueName'),
                       ksp2=GetAtt(inbound2, 'QueueName'))
```

Suggest indenting this in a way that more accurately reflects the semantic level grouping of things.

Reply:
That was autopep8 and I didn't notice it. I think switching to black will be a good idea. It's jarring at first, but it's consistent and I hate getting tied up thinking about how best to format things. Python has pretty thorough standards around it, even if I don't like all of them.